scripts/run_simavr_cycle_test.sh

```
@@ -0,0 +1,151 @@
+            }
+            printf "%s %s %s %s %s\n", seen[1], seen[2], seen[3], seen[4], seen[5];
+        }
+    ' "$vcd"
+}
+
+timestamps=$(extract_phase_timestamps) || {
+    cat "$log" >&2 || true
+    fail "did not observe a complete phase_marker trace in '$vcd'"
+}
+
+read -r t1 t2 t3 t4 t5 <<< "$timestamps"
```

Shellcheck (0.11.0)

[warning] t5 appears unused. Verify use (or export if used externally).

(SC2034)